crates/astria-conductor/src/executor/mod.rs

```diff
@@ -458,7 +461,8 @@ impl Executor {
     ))]
     async fn execute_firm(&mut self, block: ReconstructedBlock) -> eyre::Result<()> {
         let celestia_height = block.celestia_height;
-        let executable_block = ExecutableBlock::from_reconstructed(block);
+        let executable_block = ExecutableBlock::from_reconstructed(block)
+            .wrap_err("failed to construct executable block")?;
```

Comment:
This should not fail. Reconstructed blocks should be completely valid blocks. Nothing that reaches the execution task should be invalid.

Reply:
updated to be infallible - no oracle data is added to the txs if it can't be constructed.